glyphs-reader/src/font.rs

```diff
@@ -75,6 +75,7 @@ pub struct Font {
     pub virtual_masters: Vec<BTreeMap<String, OrderedFloat<f64>>>,
     pub features: Vec<FeatureSnippet>,
     pub names: BTreeMap<String, String>,
+    pub localized_names: BTreeMap<String, Vec<(String, String)>>,
```

Comment:
Suggest a comment or two about the relationship between names and localized names. Immediate questions that pop into mind for me:

* is `names` the english subset of `localized names` (equivalent to localized_names['some key']`?)
* Why is the type of `names` not the same as the value type for `localized names`?

Reply:
i'm not super happy about the distinction between bare `names` and `localized_names` myself, I did that initially to keep diffs to a minumum but I'll see if we can have one map to for all the names, English or not

Reply:
I unified the two maps in https://github.com/googlefonts/fontc/pull/1705/commits/5542ca367136c2ed4dc98e11217436114f25b73a